guides/v2.1/frontend-dev-guide/css-topics/css_debug.md

```diff
@@ -76,6 +76,7 @@ The following shows which Grunt tasks to use for debugging:
 -   After you switch the compilation mode from client-side to server-side, run the `exec` command.
 -   After you customize the content of any `.less` file, except the root source files, run the `less` task and reload the page.
 -   After you [customize the root source files or move the files included to the root files]({{site.baseurl}}/guides/v2.2/frontend-dev-guide/css-topics/css-preprocess.html#css_exception), run the `exec` command and reload the page.
+-   After run `exec` command `clear` Magento `cache`, then run `watch` command. (It will help When we run exec some of the added custom jQuery not loading like product sliders, banners etc)
```

Comment:
Can you confirm this rewrite is still technically accurate?: "After you run the `exec` command, run the `clear` command to clear the Magento `cache`, then run the `watch` command. Running the commands in this order will ensure that any custom jQuery attributes like product sliders, banners, etc are loaded correctly."